scripts/amd/gemm/tune_gemm.py

```diff
@@ -571,6 +671,9 @@ def parse_args():
     parser.add_argument("--jobs", type=int, default=1, help="number of generated files")
     parser.add_argument("--iters", type=int, default=1000, help="number of generated files")
     parser.add_argument("--init_type", type=str, default='randn', help="Initialization type for input matrices (default uniform rand [0, 1.0)])")
+    parser.add_argument("--rotating_tensor", type=int, default=0, help="total size of all tensors (default 0MB (no rotating tensor), need to be larger than the L1, L2, MALL size)")
```

Comment:
You want to add the unit of this parameter is MB

Reply:
refined